drive/drive.h

```diff
@@ -0,0 +1,28 @@
+#ifndef drive_h
+  #define drive_h
+    
+  #pragma once
+  #include "motor.h"
+  #include "Arduino.h"
+  
+  class drive {
```

Comment:
The constants could be added as public members in a `Constants.h` file and used directly in the init function.